src/components/SimpleLink.vue

```diff
@@ -8,22 +12,32 @@
 export default {
   name: 'SimpleLink',
   props: {
-    target: {
+    href: {
       type: String,
       required: true
+    },
```

Comment:
Target is used for how the browser should handle loading the link. New tab, same tab, etc.

Reply:
Aah okay! Ty!!